### PR TITLE
sig-docs: update member list

### PR DESCRIPTION
**Pull request by ran-huang**
Update the Docs SIG member list.

### CODE REVIEW

File: special-interest-groups/sig-docs/member-list.md [hunk 1]
```diff
@@ -312,3 +310,8 @@
 - [Reminiscent](https://github.com/Reminiscent)
 - [xiaoyangwang5](https://github.com/xiaoyangwang5)
 - [yuzhibotao](https://github.com/yuzhibotao)
+- [Anastasia-Lu](https://github.com/Anastasia-Lu)
```
Comment (lilin90) [inline special-interest-groups/sig-docs/member-list.md:313]: No merged PR yet by this potential contributor?

Comment (lilin90) [inline special-interest-groups/sig-docs/member-list.md:313]:
```suggestion
```

File: special-interest-groups/sig-docs/member-list.md [hunk 2]
```diff
@@ -312,3 +310,8 @@
...
+- [HuangPingZhong](https://github.com/HuangPingZhong)
+- [huangxiuyan](https://github.com/huangxiuyan)
+- [kolbe](https://github.com/kolbe)
+- [niezefeng](https://github.com/niezefeng)
```
Comment (lilin90) [inline special-interest-groups/sig-docs/member-list.md:317]: I didn't see merged PR in four doc repos by this potential contributor. Please help confirm.

**Comment by ran-huang** [comment]
@lilin90 @yikeke PTAL again. Thanks! Made two changes:
- Remove IDs that only reviewed PRs but never merged any;
- Put each list in alphabetical order.

File: special-interest-groups/sig-docs/member-list.md [hunk 3]
```diff
@@ -12,303 +12,270 @@
...
 
 - [anotherrachel](https://github.com/anotherrachel)
 - [aylei](https://github.com/aylei)
-- [YiniXu9506](https://github.com/YiniXu9506)
-- [weekface](https://github.com/weekface)
 - [crazycs520](https://github.com/crazycs520)
-- [juliezhang1112](https://github.com/juliezhang1112)
+- [csuzhangxc](https://github.com/csuzhangxc)
 - [ericsyh](https://github.com/ericsyh)
+- [juliezhang1112](https://github.com/juliezhang1112)
```
Comment (lilin90) [inline special-interest-groups/sig-docs/member-list.md:34]: This ID has changed, so now this link is broken. The new ID is `junlan-zhang`.
```suggestion
- [juliezhang1112](https://github.com/junlan-zhang)
```

**Pull request merged**